serverless-api.yml

```diff
@@ -63,6 +63,8 @@ functions:
       VULNERABILITIES_BASE_URL: ${ssm:/hn-single-view-api/${self:provider.stage}/VULNERABILITIES_BASE_URL}
       ACADEMY_API_BASE_URL: ${ssm:/hn-single-view-api/${self:provider.stage}/ACADEMY_API_BASE_URL}
       ACADEMY_API_API_KEY: ${ssm:/hn-single-view-api/${self:provider.stage}/ACADEMY_API_API_KEY}
+      CAUTIONARY_ALERTS_BASE_URL: ${ssm:/hn-single-view-api/${self:provider.stage}/CAUTIONARY_ALERTS_BASE_URL}
+      CAUTIONARY_ALERTS_API_KEY: ${ssm:/hn-single-view-api/${self:provider.stage}/CAUTIONARY_ALERTS_API_KEY}
```

Comment:
In case you weren't aware, these also need to be added to aws system manager parameter store. In the old aws account.

Reply:
Thanks!